### ISSUE TITLE
Cluster names in the titles of error e-mails

### ISSUE THREAD

**Issue by cameel** [golemfactory/concent-deployment#39]
Currently the titles of e-mails with error reports look like this for all clusters:

```
[Django] ERROR (EXTERNAL IP): Internal Server Error: /api/v1/send/
```

It would be better if they were different for different clusters:

```
[concent-api-dev] ERROR (EXTERNAL IP): Internal Server Error: /api/v1/send/
```
```
[concent-api-staging] ERROR (EXTERNAL IP): Internal Server Error: /api/v1/send/
```
```
[gatekeeper-dev] ERROR (EXTERNAL IP): Internal Server Error: /download/
```
```
[gatekeeper-staging] ERROR (EXTERNAL IP): Internal Server Error: /download/
```

The name should identify both the cluster and the container that's running Django.

Use [EMAIL_SUBJECT_PREFIX](https://docs.djangoproject.com/en/1.11/ref/settings/#email-subject-prefix) setting.

**Comment by cameel** [comment]
I have an idea that will let us reuse current variables. Let's do this:

```
[concent-dev:concent-api] ERROR (EXTERNAL IP): Internal Server Error: /api/v1/send/
```
```
[concent-staging:gatekeeper] ERROR (EXTERNAL IP): Internal Server Error: /download/
```

**Comment by cameel** [comment]
Merged into master: 6b38f42113a3573045da6239ab2ca32332e74062
